internal/db/rdb/mysql/option_test.go

```diff
@@ -19,1704 +19,1131 @@ package mysql
-			   obj := new(T)
-			   got(obj)
-			   if err := test.checkFunc(tt.want, obj); err != nil {
-			       tt.Errorf("error = %v", err)
-			   }
-			*/
+			if test.checkFunc == nil {
+				test.checkFunc = defaultCheckFunc
+			}
+
+			got := WithMaxOpenConns(test.args.conns)
+			obj := new(T)
+			if err := test.checkFunc(test.want, obj, got(obj)); err != nil {
+				tt.Errorf("error = %v", err)
+			}
 		})
 	}
 }
 
 func TestWithTLSConfig(t *testing.T) {
-	type T = interface{}
+	type T = mySQLClient
 	type args struct {
 		cfg *tls.Config
 	}
 	type want struct {
 		obj *T
-		// Uncomment this line if the option returns an error, otherwise delete it
-		// err error
+		err error
 	}
 	type test struct {
-		name string
-		args args
-		want want
-		// Use the first line if the option returns an error. otherwise use the second line
-		// checkFunc  func(want, *T, error) error
-		// checkFunc  func(want, *T) error
+		name       string
+		args       args
+		want       want
+		checkFunc  func(want, *T, error) error
 		beforeFunc func(args)
 		afterFunc  func(args)
 	}
 
-	// Uncomment this block if the option returns an error, otherwise delete it
-	/*
-	   defaultCheckFunc := func(w want, obj *T, err error) error {
-	       if !errors.Is(err, w.err) {
-	           return errors.Errorf("got error = %v, want %v", err, w.err)
-	       }
-	       if !reflect.DeepEqual(obj, w.obj) {
-	           return errors.Errorf("got = %v, want %v", obj, w.obj)
-	       }
-	       return nil
-	   }
-	*/
-
-	// Uncomment this block if the option do not returns an error, otherwise delete it
-	/*
-	   defaultCheckFunc := func(w want, obj *T) error {
-	       if !reflect.DeepEqual(obj, w.obj) {
-	           return errors.Errorf("got = %v, want %v", obj, w.c)
-	       }
-	       return nil
-	   }
-	*/
+	defaultCheckFunc := func(w want, obj *T, err error) error {
+		if !errors.Is(err, w.err) {
+			return errors.Errorf("got error = %v, want %v", err, w.err)
+		}
+		if !reflect.DeepEqual(obj, w.obj) {
+			return errors.Errorf("got = %v, want %v", obj, w.obj)
+		}
+		return nil
+	}
 
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           cfg: nil,
-		       },
-		       want: want {
-		           obj: new(T),
-		       },
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           cfg: nil,
-		           },
-		           want: want {
-		               obj: new(T),
-		           },
-		       }
-		   }(),
-		*/
+		func() test {
+			cfg := new(tls.Config)
+			return test{
+				name: "set success when cfg is not nil",
+				args: args{
+					cfg: cfg,
+				},
+				want: want{
+					obj: &T{
+						tlsConfig: cfg,
+					},
+				},
+			}
+		}(),
+		func() test {
+			return test{
+				name: "set success when cfg is nil",
+				want: want{
+					obj: new(T),
+				},
+			}
+		}(),
```

Comment:
```suggestion
		{
			name: "set success when cfg is nil",
			want: want{
				obj: new(T),
			},
		},
```

Reply:
I'd not like to contain some different formats for readability 😕 
How do you think? I discussed about this before with @hlts2

Reply:
@vankichi I think my suggestion is easier to read (the default format of table driven test) so I suggested here. 
Both fine for me :)
@hlt2 what do you think?

Reply:
@vankichi @kevindiu 
I discussed about this before with @vankichi 
I agreed to unify the formats.
And I recently thought that the unification improved the readability.

Reply:
both fine for me :) will resolve the conversation